docs/can-guides/commitment/recipes/file-navigator/file-navigator-simple.md

```diff
@@ -374,9 +372,9 @@ Component.extend({
   tag: "file-navigator",
   view: `
     {{<entities}}
-      <span>{{this.name}}</span>
+      <span>{{rootEntity.name}}</span>
       <ul>
-        {{# for(child of this.children) }}
+        {{# for(child of rootEntity.name) }}
```

Comment:
This doesn’t look right. Did you run through the guide to make sure all the steps were right / worked?

Reply:
I updated, yeah I did it but it seems missed this step.